pootle/apps/pootle_app/management/commands/clear_stats.py

```diff
@@ -12,11 +12,42 @@
 # This must be run before importing Django.
 os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
 
-from pootle_app.management.commands import PootleCommand
+from django.core.management.base import NoArgsCommand
 
+from django_redis import get_redis_connection
+from redis import WatchError
 
-class Command(PootleCommand):
```

Comment:
If this no longer extends `PootleCommand` we are loosing the ability to clear the stats for a specific language, or project, or TP. Is that the desired behavior?